lerna.json

```diff
@@ -1,5 +1,12 @@
 {
 	"lerna": "2.8.0",
+	"commands": {
+		"publish": {
+			"ignore": [
+				"**/test/*.js"
```

Comment:
I want to test if that excludes such files from publishing to npm or this is only used to detect changed files when deciding if bump version. If the latter I will investigate how to prevent those files from being published in a different way.

Reply:
It is only useful to prevent publishing a new version when no code related changes are applied. See what gets downloaded from the npm:

![screen shot 2018-02-22 at 13 02 06](https://user-images.githubusercontent.com/699132/36537533-b9bbcdea-17d0-11e8-85b3-044819fcea3c.png)

`test` folder is there. I'm wondering if there is a different way to stop publishing test file. We can always move code `src` and let build script do the file filtering.

Reply:
That would work 👍